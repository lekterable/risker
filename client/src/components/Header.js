import React, { Component } from 'react'
import { NavLink } from 'react-router-dom'
import './Header.css'

export default class extends Component {
	render() {
		return (
			<nav className="row navbar navbar-expand-md navbar-light bg-light">
				<NavLink className="navbar-brand" to="/">
					<span role="img" aria-label="dice">
						🎲
					</span>{' '}
					risker app
				</NavLink>
				<button
					className="navbar-toggler"
					type="button"
					data-toggle="collapse"
					data-target="#navbarColor03"
					aria-controls="navbarColor03"
					aria-expanded="false"
					aria-label="Toggle navigation"
				>
					<span className="navbar-toggler-icon" />
				</button>
				<div className="collapse navbar-collapse" id="navbarColor03">
					<ul className="navbar-nav">
						<li className="nav-item">
							<NavLink exact className="nav-link" to="/game">
								<span role="img" aria-label="loupe">
									🕹
								</span>{' '}
								game
							</NavLink>
						</li>
						<li className="nav-item">
							<NavLink exact className="nav-link" to="/ranking">
								<span role="img" aria-label="plus">
									📈
								</span>{' '}
								ranking
							</NavLink>
						</li>
					</ul>
				</div>
			</nav>
		)
	}
}
