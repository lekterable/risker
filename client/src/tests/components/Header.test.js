import React from 'react'
import { shallow } from 'enzyme'
import Header from '../../components/Header'

describe('Header', () => {
	it('should match snapshot', () => {
		const wrapper = shallow(<Header />)

		expect(wrapper).toMatchSnapshot()
	})
})
