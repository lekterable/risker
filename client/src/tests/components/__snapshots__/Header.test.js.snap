// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<nav
  className="row navbar navbar-expand-md navbar-light bg-light"
>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    className="navbar-brand"
    to="/"
  >
    <span
      aria-label="dice"
      role="img"
    >
      ðŸŽ²
    </span>
     
    risker app
  </NavLink>
  <button
    aria-controls="navbarColor03"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarColor03"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarColor03"
  >
    <ul
      className="navbar-nav"
    >
      <li
        className="nav-item"
      >
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="nav-link"
          exact={true}
          to="/game"
        >
          <span
            aria-label="loupe"
            role="img"
          >
            ðŸ•¹
          </span>
           
          game
        </NavLink>
      </li>
      <li
        className="nav-item"
      >
        <NavLink
          activeClassName="active"
          ariaCurrent="true"
          className="nav-link"
          exact={true}
          to="/ranking"
        >
          <span
            aria-label="plus"
            role="img"
          >
            ðŸ“ˆ
          </span>
           
          ranking
        </NavLink>
      </li>
    </ul>
  </div>
</nav>
`;
